### PR TITLE
Add: Tooltip to TVL Breakdown page 

### PR DESCRIPTION
**Pull request by Gathin23**
## Implementation
This implementation gives the working of tooltip in both light and dark mode colours and the mobile view

Screenshot_20231015_210026
Screenshot_20231015_205921

## Solves
#2035

### CODE REVIEW

**Review by torztomasz — CHANGES_REQUESTED** [review]
Hey! Your PR looks good. Fix two issues mentioned in comment and we can merge it :)

File: packages/frontend/src/components/tvl-breakdown/TvlBreakdownSummaryBox.tsx [hunk 1]
```diff
@@ -54,19 +60,29 @@ function StatsItem(props: StatsItemProps) {
...
+        
+        >
+          {props.mobileTitle}
+        </span>
+      
+        <span
+          className="Tooltip -translate-y-px md:translate-y-0 ml-0.5"
+          title={props.tooltip}
+        >
+          <InfoIcon className="fill-current md:h-3.5 md:w-3.5" />
```
[the remark this answers is not in the document]
Comment (Gathin23) [reply packages/frontend/src/components/tvl-breakdown/TvlBreakdownSummaryBox.tsx]: I removed the fill-current and now the fill be of the same colour as the text as the default colour of the tooltip is same as the text colour used

**Comment by Gathin23** [comment]
Did the necessary changes @torztomasz

**Comment by torztomasz** [comment]
## Desktop
image
Tooltip background color does not match the text

## Mobile
image
Tooltip only matches color with the top one


You can check your implementation on a [preview URL](https://l2beat-staging-ey7nh9ru3-l2beat.vercel.app/scaling/projects/arbitrum/tvl-breakdown)

**Comment by Gathin23** [comment]
Got it! Will make the necessary changes now @torztomasz

**Comment by Gathin23** [comment]
Made the necessary changes @torztomasz 

## Screenshots

Screenshot_20231018_185442
Screenshot_20231018_185505
Screenshot_20231018_185818

File: packages/frontend/src/components/tvl-breakdown/TvlBreakdownSummaryBox.tsx [hunk 2]
```diff
@@ -11,25 +12,29 @@ export function TvlBreakdownSummaryBox(
...
       <StatsItem
         title="Externally Bridged"
+        tooltip="Total value of externally bridged tokens displayed together with a percentage change compared to 7D ago."
         mobileTitle="Externally Bridged Value"
         value={props.external.value}
         change={props.external.change}
       />
       <StatsItem
         title="Canonically Bridged"
+        tooltip="Total value of natively minted tokens displayed together with a percentage change compared to 7D ago."
```
Comment (torztomasz) [inline packages/frontend/src/components/tvl-breakdown/TvlBreakdownSummaryBox.tsx:37]: Wrong tooltip content

image

**Review by torztomasz — APPROVED** [review]

**Comment by Gathin23** [comment]
A careless copy typo which is now fixed! @torztomasz

**Review by torztomasz — APPROVED** [review]

**Pull request merged**